lib/common/fse_decompress.c

```diff
@@ -184,16 +184,22 @@ FORCE_INLINE_TEMPLATE size_t FSE_decompress_usingDTable_generic(
     FSE_DState_t state1;
     FSE_DState_t state2;
 
+    BIT_DStream_status reloadStat;
```

Comment:
If it was a hot portion of the code, I would be concerned by defining a variable outside of the loop, as it makes compiler optimizations a bit more difficult.
But since this code is only used to decompress Huffman weights, I presume any speed different is unlikely to be noticeable.

Reply:
From an optimization standpoint, it was more about avoiding a redundant extra call to `BIT_reloadDStream` (since I don't think the compiler would be able to reliably tell that calling it multiple times consecutively would be a no-op) but I guess I'll change it back.